corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -1012,7 +1012,7 @@ hqDefine('geospatial/js/models', [
             let caseIdToOwnerId = {};
             for (const caseItem of self.mapModel.caseMapItems()) {
                 const caseData = self.mapModel.caseGroupsIndex[caseItem.itemId];
-                if (caseData.assignedUserId) {
+                if ('assignedUserId' in caseData && caseData.assignedUserId) {
```

Comment:
This was required because  I ran into a situation with cases having no user assigned to them after the disbursement based on the cases selection.  

I figured having this check is safe and does no harm. 

Looking more , also it looks like the current disbursement does not works correctly with the saved areas feature.
I am going to create a separate  ticket for this.

Reply:
okay, so if there is no user assigned isn't `caseData.assignedUserId` `undefined`? I am unsure why the check `'assignedUserId' in caseData` needed.